_episodes/046-alice-ryhl.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Tokio Ecosystem with Alice Ryhl"
+date: 2021-11-12T02:11:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e046-alice-ryhl.mp3
+duration: "1:02:51"
+length: "61148368"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Alice Ryhl](https://ryhl.io/), one of the primary maintainer of the open source project [Tokio](https://tokio.rs/), a widely used Rust library.
+
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps 
+- [@00:40] - Alice's Bio
+- [@01:08] - Pull request's process in GitHub
+- [@05:00] - Alice's involvement in Tokio
+- [@08:23] - Tokio's topics page
+- [@11:06] - Alice's favorite part in the contribution
+- [@12:55] - Changes in Tokio's since Alice joined
+- [@16:52] - Measuring metrics
+- [@19:38] - Cooperative scheduling
+- [@22:52] - Cooperative & Preemptive scheduler 
+- [@24:30] - [Diesel](https://diesel.rs/)
+- [@25:45] - Definition of [Blocking]((https://ryhl.io/blog/async-what-is-blocking/)
+- [@27:37] - I/O threads
+- [@31:21] - Meaning of sleeping threads
+- [@33:41] - [Tokio Console](https://tokio.rs/blog/2021-09-console-dev-diary-1)
+- [@39:07] - Constructs, Actors, Actics within Tokio 
+- [@41:14] - Pros and Cons of Actor patterns
+- [@47:05] - Alice's academic background
+- [@49:22] - [Tokio upcoming roadmap](https://tokio.rs/blog/2020-12-tokio-1-0)
```

Comment:
The link here is weird to me, since it's describing 1.0, which has already launched, so it's not really a roadmap?